content/workflow.md

```diff
@@ -0,0 +1,406 @@
+---
+id: workflow
+title: Workflow
+sidebar_label: Workflow
+slug: https://docs.hypi.io/docs/workflow
```

Comment:
The example I gave was exactly what you needed. Don't put the http..., just `slug: /workflow` the documentation framework will turn this into `https://docs.hypi.io/docs/workflow`

Reply:
corrected